### PR TITLE
Fix NPE where the country code isn't specified in the sellAsset of Sep38Service

### PR DESCRIPTION
**Pull request by Ifropc**
<details>
  <summary>PR Checklist</summary>

### PR Structure

* [ ] This PR has reasonably narrow scope (if not, break it down into smaller PRs).
* [ ] This PR avoids mixing refactoring changes with feature changes (split into two PRs
  otherwise).
* [ ] This PR's title starts with name of package that is most changed in the PR, ex.
  `paymentservice.stellar`, or `all` or `doc` if the changes are broad or impact many
  packages.

### Thoroughness

* [ ] This PR adds tests for the most critical parts of the new functionality or fixes.
</details>

### What

fix npe

### Why

fix npe

### Known limitations

n/a

### CODE REVIEW

**Review by lijamie98 — APPROVED** [review]
LGTM

File: core/src/main/java/org/stellar/anchor/sep38/Sep38Service.java [hunk 1]
```diff
@@ -89,7 +89,8 @@ public GetPricesResponse getPrices(
 
     // countryCode
     if (!Objects.toString(countryCode, "").isEmpty()) {
-      if (!sellAsset.getCountryCodes().contains(countryCode)) {
+      if (sellAsset.getCountryCodes() == null
+          || !sellAsset.getCountryCodes().contains(countryCode)) {
         throw new BadRequestException("Unsupported country code");
```
Comment (lijamie98) [inline core/src/main/java/org/stellar/anchor/sep38/Sep38Service.java:94]: The country code isn't specified or is specified but not supported.

**Review by JakeUrban — APPROVED** [review]
Is this only relevant to `getPrices()` or are there other places this should be checked?

**Comment by Ifropc** [comment]
All other places check that it's not null

**Pull request merged**